jetty-webapp/src/main/java/org/eclipse/jetty/webapp/WebAppContext.java

```diff
@@ -1434,7 +1434,7 @@ protected void stopContext() throws Exception
                 setClassLoader(null);
             }
 
-            setAvailable(true);
```

Comment:
Need an explanation of this change as on the face of it, it is the exact opposite of the original intent.

Reply:
I can't really explain.  It looks to me that it was just plain wrong, but didn't matter because we were stopping already.
I think it actually dates from when availability was just in the webapp and it had to clear unavailable status   This is no longer required, so I will try without anything here.